docs/en-US/component/tree-v2.md

```diff
@@ -124,6 +124,7 @@ tree-v2/filter
 | Name             | Description                                          | Parameters                                                                                                                              |
 | ---------------- | ---------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
 | node-click       | triggers when a node is clicked                      | `(data: TreeNodeData, node: TreeNode, e: MouseEvent)`                                                                                   |
+| node-drop        | triggers when drag someting and drop on a node       | `(data: TreeNodeData, node: TreeNode, e: DragEvent)`                                                                                    |
```

Comment:
Need to add the version number here.
^(2.8.2)

Reply:
OK

Reply:
Sorry, I misread it, it should be 2.8.3